Sources/TuistGenerator/Models/Target.swift

```diff
@@ -100,10 +100,10 @@ public class Target: Equatable, Hashable {
         }
     }
 
-    public static func sources(projectPath: AbsolutePath, sources: [(glob: String, compilerFlags: String?)]) throws -> [Target.SourceFile] {
+    public static func sources(projectPath _: AbsolutePath, sources: [(glob: String, compilerFlags: String?)]) throws -> [Target.SourceFile] {
         var sourceFiles: [AbsolutePath: Target.SourceFile] = [:]
         sources.forEach { source in
-            projectPath.glob(source.glob).filter { path in
+            AbsolutePath("/").glob(String(source.glob.dropFirst())).filter { path in
```

Comment:
As a suggestion this could be:

```swift
let sourcePath = AbsolutePath(source.glob)
let base = AbsolutePath(sourcePath.dirname)
base.glob(sourcePath.basename)
```

Not sure if its any clearer